### PR TITLE
Add package scripts to package.json

### PR DESCRIPTION
**Pull request by lygstate**
So the developer can package vscode-cmake-tools easily.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1783,7 +1784,8 @@
     "tslint": "^5.20.1",
     "typedoc": "^0.20.25",
     "typescript": "^4.1.5",
-    "vscode": "^1.1.36",
+    "vsce": "^1.87.1",
+    "vscode": "^1.1.37",
```
Comment (bobbrow) [inline package.json:1788]: We typically don't like to upgrade the vscode version unless we have to.  We want to be able to run on the lowest possible version.

Comment (lygstate) [reply package.json:1788]: Oh, sorry for that, I upgrade that because when I do yarn install it's failed locally

Comment (bobbrow) [reply package.json:1788]: Are you saying that this change requires the update to 1.1.37?

Comment (lygstate) [reply package.json]: That's not necessary, fixed

**Review by bobbrow — APPROVED** [review]

**Pull request merged**